clarifai/client/input.py

```diff
@@ -541,23 +572,43 @@ def get_mask_proto(input_id: str, label: str, polygons: List[List[float]],
     """
     if not isinstance(polygons, list):
       raise UserError("polygons must be a list of points")
-    input_mask_proto = resources_pb2.Annotation(
-        input_id=input_id,
-        data=resources_pb2.Data(regions=[
-            resources_pb2.Region(
-                region_info=resources_pb2.RegionInfo(polygon=resources_pb2.Polygon(
-                    points=[
-                        resources_pb2.Point(
-                            row=_point[1],  # row is y point
-                            col=_point[0],  # col is x point
-                            visibility="VISIBLE") for _point in polygons
-                    ])),
-                data=resources_pb2.Data(concepts=[
-                    resources_pb2.Concept(
-                        id=f"id-{''.join(label.split(' '))}", name=label, value=1.)
-                    if not label_id else resources_pb2.Concept(id=label_id, name=label, value=1.)
-                ]))
-        ]))
+    if annot_id:
+      input_mask_proto = resources_pb2.Annotation(
+          id=annot_id,
```

Comment:
Same as the `get_bbox_proto` function comment!

Reply:
Done